app/views/users/index.html.erb

```diff
@@ -0,0 +1,11 @@
+<h4>List of all users...</h4>
+
+<div id='card'>
+  <% @users.each do |user| %>
+    <img src=<%=user.photo%> alt=<%=user.name%> id='card_img'>
+    <div id = 'card_text'>
+      <h2><%= user.name %> </h2> 
+      <p> Number of posts: <%= user.posts_counter %></p>
+    </div>
+  <% end %>
+</div>
```

Comment:
- [ ] Please make sure clicking on each user takes you to the user show page. You can use the `link_to` method as explained [here](https://www.rubyguides.com/2019/05/rails-link_to-method/).